src/content_mapblock.cpp

```diff
@@ -172,6 +172,10 @@ static inline void getNeighborConnectingFace(v3s16 p, INodeDefManager *nodedef,
 		*neighbors |= v;
 }
 
+static int NeighbourToIndex(const v3s16 &pos) {
```

Comment:
Marking this function as inline probably won't have any effect on the compiler's decision.  What it will do is signal the author's intent on the intended role of this function.  So, I agree with Zeno` that maybe it should be marked inline. 
Also I think this function should be named "MeshGenerateSpecialNeighbourToIndex" to indicate its relation to that function too.